web/ui/src/Pools/PoolsController.js

```diff
@@ -94,20 +122,44 @@
             });
         };
 
-        // Function for adding new pools - through modal
-        $scope.add_pool = function() {
-            return resourcesFactory.addPool($scope.newPool)
-                .success(function(data){
-                    poolsFactory.update();
-                    // Reset for another add
-                    $scope.newPool = {};
-                });
-        };
-
         $scope.isDefaultPool = function(poolID) {
           return poolID === "default";
         };
 
+        function updatePools(){
+            resourcesFactory.getV2Pools()
+                .success(data => {
+                    $scope.pools = data.results.map(result => { return new Pool(result); });
```

Comment:
super itty bitty thing, arrow functions dont need brackets and have an implicit return, so you can more succinctly write

`data.results.map(result => new Pool(result));`